<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager</class>
 <widget class="QWidget" name="manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>592</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plugman</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="2" column="0" rowspan="6" colspan="6">
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item row="8" column="0" colspan="6">
    <widget class="QProgressBar" name="progressBar">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="value">
      <number>53</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QLineEdit" name="findField">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>Not yet implemented</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="5">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QPushButton" name="actionsButton">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Actions</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/internet.png</normaloff>:/icons/internet.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="findButton">
     <property name="maximumSize">
      <size>
       <width>96</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>find</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/find.png</normaloff>:/icons/find.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
